packages/graphicsmagick.rb

```diff
@@ -3,60 +3,122 @@
   homepage 'http://www.graphicsmagick.org/'
-  version '1.3.43-icu75.1'
+  version '1.3.45-icu75.1'
   license 'MIT'
-  compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.43/GraphicsMagick-1.3.43.tar.xz'
-  source_sha256 '2b88580732cd7e409d9e22c6116238bef4ae06fcda11451bf33d259f9cbf399f'
+  compatibility 'all'
+  min_glibc (%w[x86_64 aarch64 armv7l].include?(ARCH) ? '2.37' : '2.23').to_s
+  source_url 'https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.45/GraphicsMagick-1.3.45.tar.xz'
+  source_sha256 'dcea5167414f7c805557de2d7a47a9b3147bcbf617b91f5f0f4afe5e6543026b'
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'aeff59b26a27af04fd89485d29b4375be3b526eaebef964a22ec8c027082467e',
-     armv7l: 'aeff59b26a27af04fd89485d29b4375be3b526eaebef964a22ec8c027082467e',
-     x86_64: 'd10dba42f7cdf317ef437c4fdd6500d7cce82eb1fc56e9596e1a494f3b13cda1'
+       i686: '3e03732df5107a6c8617b785baae64d477ead57553087fafa31de0822d7f06fc',
+    aarch64: 'd43124fbe8d5b9d83baee6a6f414de0cfebf3c8ece98b988bb1e4b15af70bb7d',
+     armv7l: 'd43124fbe8d5b9d83baee6a6f414de0cfebf3c8ece98b988bb1e4b15af70bb7d',
+     x86_64: '36081e952cf6c73855cf4b59770bf5b89ecfed1fe9f4f9a16105d8ebbd5b8113'
   })
 
   depends_on 'brotli' # R
   depends_on 'bzip2' # R
-  depends_on 'freetype' # R
+  depends_on 'freetype' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'gcc_lib' # R
-  depends_on 'ghostscript' => :build
+  depends_on 'ghostscript' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'glibc' # R
-  depends_on 'harfbuzz' # R
+  depends_on 'harfbuzz' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'highway' # R
   depends_on 'icu4c' # R
-  depends_on 'jasper' # R
+  depends_on 'jasper' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'jbigkit' # R
   depends_on 'lcms' # R
   depends_on 'libbsd' # R
-  depends_on 'libde265' # R
+  depends_on 'libde265' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libdeflate' # R
-  depends_on 'libheif' # R
+  depends_on 'libheif' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libice' # R
   depends_on 'libjpeg_turbo' # R
-  depends_on 'libjxl' # R
+  depends_on 'libjxl' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libmd' # R
   depends_on 'libpng' # R
-  depends_on 'libsm' # R
+  depends_on 'libsm' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libtiff' # R
   depends_on 'libtool' # R
-  depends_on 'libwebp' # R
-  depends_on 'libwmf' # R
-  depends_on 'libx11' # R
+  depends_on 'libwebp' if %w[x86_64 aarch64 armv7l].include?(ARCH)
+  depends_on 'libwmf' if %w[x86_64 aarch64 armv7l].include?(ARCH)
+  depends_on 'libx11' if %w[x86_64 aarch64 armv7l].include?(ARCH)
   depends_on 'libxau' # R
   depends_on 'libxcb' # R
   depends_on 'libxdmcp' # R
-  depends_on 'libxext' # R
+  depends_on 'libxext' if %w[x86_64 aarch64 armv7l].include?(ARCH)
```

Comment:
Could all these `depends_on ...` lines be inside one conditional?
```
if %w[x86_64 aarch64 armv7l].include?(ARCH)
  depends_on ....
end
```

Reply:
Done.